internal/factory/container/impl.go

```diff
@@ -0,0 +1,6 @@
+package container
+
+// Impl is the main implementation interface of this package.
+type Impl interface {
+	SecurityLabel(path, secLabel string, shared, maybeRelabel bool) error
+}
```

Comment:
I don't think this is needed if you just add a `label_unsupported.go` file that defines SecurityLabel as an empty function